### PR TITLE
Add documentation to activate HashiCorp Flex

### PR DESCRIPTION
**Pull request by BrianMMcClain**
🔗 [Deploy Preview](https://terraform-docs-common-git-bmm-activate-hashicorp-flex-hashicorp.vercel.app/terraform/cloud-docs/overview/activate-flex)

### What
Add documentation to activate HashiCorp Flex

### Why
With the latest changes to Terraform Cloud pricing, we want to provide instructions on how users can activate HashiCorp flex

----------

### Merge Checklist
_If items do not apply to your changes, add (N/A) and mark them as complete._

#### Pull Request
- [x] One or more labels describe the type of change (e.g. clarification) and associated product (e.g. tfc).
- [x] Description links to related pull requests or issues, if any.

#### Content
- [x] Redirects have been added to `website/redirects.js` for moved, renamed, or deleted pages.
- [x] API documentation and the API Changelog have been updated. 
- [x] Links to related content where appropriate (e.g., API endpoints, permissions, etc.).
- [x] Pages with related content are updated and link to this content when appropriate.
- [x] Sidebar navigation files have been updated for added, deleted, reordered, or renamed pages.
- [x] New pages have metadata (page name and description) at the top.
- [x] New images are 2048 px wide. They have HashiCorp standard annotation color (#F92672) and format (rectangle with rounded corners), blurred sensitive details (e.g. credentials, usernames, user icons), and descriptive alt text in the markdown for accessibility.
- [x] New code blocks have the correct syntax and line breaks to eliminate horizontal scroll bars.
- [x] UI elements (button names, page names, etc.) are bolded.
- [x] The Vercel website preview successfully deployed.

#### Reviews
- [x] I or someone else reviewed the content for technical accuracy.
- [x] I or someone else reviewed the content for typos, punctuation, spelling, and grammar.

### CODE REVIEW

**Review by leizerbeam — APPROVED** [review]
+1 approval!

File: website/docs/cloud-docs/overview/activate-flex.mdx [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+![A view of the Terraform Cloud account summary showing billing summary and available balance](/img/docs/tfc-account-summary.png)
+
+## Step 4: Edit the Terraform Cloud Organization Plan
+
+Navigate to the TFC organization setting's **Plan & Billing** page. For TFC user accounts that are not unified with HCP user accounts, you will be asked to login to an HCP account that has the `admin` role in the HCP organization with HashiCorp Flex allocated.  This login is required to allow you to edit the TFC organization's plan.  
+
+~> **Note**: If you have a unified HCP account, you do not need to login and TFC will automatically identify qualified HCP organizations that your unified HCP user account is a member of.
+
+Click **Edit Plan** to be brought to the Plan Activation page.
```
Comment (ritsok) [inline website/docs/cloud-docs/overview/activate-flex.mdx:67]:
```suggestion
Click **Edit Plan** to go to the Plan Activation page.  
```

File: website/docs/cloud-docs/overview/activate-flex.mdx [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+## Step 4: Edit the Terraform Cloud Organization Plan
+
+Navigate to the TFC organization setting's **Plan & Billing** page. For TFC user accounts that are not unified with HCP user accounts, you will be asked to login to an HCP account that has the `admin` role in the HCP organization with HashiCorp Flex allocated.  This login is required to allow you to edit the TFC organization's plan.  
+
+~> **Note**: If you have a unified HCP account, you do not need to login and TFC will automatically identify qualified HCP organizations that your unified HCP user account is a member of.
+
+Click **Edit Plan** to be brought to the Plan Activation page.  
+
+If your HashiCorp user account has the `owner` or `admin` role in multiple HCP organizations, you will be asked to select which Billing Account to configure against.
```
Comment (ritsok) [inline website/docs/cloud-docs/overview/activate-flex.mdx:69]:
```suggestion
If your HashiCorp user account has the `owner` or `admin` role in multiple HCP organizations, you will need to select a billing account.
```

File: website/docs/cloud-docs/overview/activate-flex.mdx [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+![A view prompting the user to choose the billing account to link against](/img/docs/tfc-plan-org-select.png)
+
+After selecting a plan, confirm your selection and review the **Cost Estimate**.
+
+If you meet all the necessary [prerequisites](#prerequisites), the **Plan Selection** page will show “HashiCorp Flex” at the bottom left. If you see “Pay-as-you-go,” please review steps 1-3 or [contact HashiCorp support](https://support.hashicorp.com/hc/en-us/requests/new) for more assistance.
```
Comment (ritsok) [inline website/docs/cloud-docs/overview/activate-flex.mdx:75]: bold UI elements instead of quotes

**Pull request merged**